cdm/src/test/java/ucar/nc2/ft2/coverage/simpgeometry/TestReadPolygon.java

```diff
@@ -0,0 +1,64 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.ma2.Array;
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.nc2.ft2.coverage.simpgeometry.*;
+import ucar.unidata.util.test.TestDir;
+
+/**
+ * Polygon construction tests at the dataset level.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestReadPolygon {
+
+	/**
+	 * Tests ability to read and construct Polygons and Multipolygons.
+	 * 
+	 */
+	@Test
+	public void testMultiPolygonsPolygons() {

```

Comment:
I like this test method!

Reply:
Thanks! Perhaps this will be a model for the similar Read Point and Read Line as well.